vagrant/ansible/roles/sit.cephfs/tasks/server/centos.yml

```diff
@@ -32,5 +31,6 @@
     name:
       - podman
       - lvm2
+      - cephadm
```

Comment:
Looks like `cephadm`  is a _noarch_ package and we haven't configured it yet. Can we add it to the baseurl field?

Reply:
Hm.. multiple baseurls is not an option as they are only searched in case of failures. Therefore we could directly download the `cephadm` binary instead of installing the package as follows:

```
- name: Download cephadm
  get_url:
    url: {{ ceph_shaman.json[0].url }}/noarch/cephadm
    dest: /root/cephadm
    mode: 0755
```

Reply:
Won't this have the same potential problem of missing dependencies ?

Reply:
I don't think so. This is as per the [docs](https://docs.ceph.com/en/latest/cephadm/install/#cephadm-install-curl) @phlogistonjohn linked earlier. Only change here is that we download `cephadm` based on latest main branch and not from a release.

Reply:
That's fine. I'll update the PR.